subapp/src/utils/constants.ts

```diff
@@ -67,10 +69,15 @@ export const IS_SHOW_ASSISTANTS = {
 export const PRIMARY_100 = '#202452';
 export const DANGER_900 = '#74122E';
 
+// Roles
 export const ROLE_ASSISTANT = 'assistant';
 export const ROLE_USER = 'user';
 export const ROLE_TOOL = 'tool';
 export const ROLE_WAIT = 'wait';
 export const ROLE_NODE = 'node';
 export const ROLE_BOT = 'bot';
 export const ROLE_ERROR = 'error';
+
+// Keyboard constants
+export const KEYBOARD_BEHAVIOR = Platform.OS === 'ios' ? 'padding' : 'height';
+export const KEYBOARD_VERTICAL_OFFSET = Platform.OS === 'ios' ? Dimensions.get('window').height * 0.05 : 0;
```

Comment:
is "0.05" coming from some iOS internal constant or just a value you got based on visual tests?

Reply:
@LuuchoRocha The value 0.05 is not an internal iOS constant; it is a value that works well across a variety of screen resolutions. It was chosen based on visual testing to ensure a consistent offset when the keyboard is displayed, providing a more adaptable and generic solution